bentoml/saved_bundle/templates.py

```diff
@@ -82,18 +82,19 @@
 # copy over entrypoint scripts
 COPY docker-entrypoint.sh /usr/local/bin/
 
+# Copy environment.yml, because bundled_pip_dependencies might not exist. This
+# prevent COPY command from failing.
+COPY environment.yml bundled_pip_dependencies*  /bento/bundled_pip_dependencies/
+
+# Remove environment.yml from bundled_pip_dependencies directory
+RUN rm /bento/bundled_pip_dependencies/environment.yml
```

Comment:
Remove the extra environment.yml from `bundled_pip_dependencies` directory, so `bentoml-init.sh` won't throw error when it try to do `pip install environment.yml`.